speed: 8.0
radius: 6.0
ramp_length: 25.0
